### PR TITLE
Integrated django-spaghetti-and-meatballs to visualize database schema.

### PR DESCRIPTION
**Pull request by RishabhJain2018**
#193 Solved. @deshraj Please review the PR.

### CODE REVIEW

File: settings/common.py [hunk 1]
```diff
@@ -155,3 +156,9 @@
 
 
 CORS_ORIGIN_ALLOW_ALL = True
+
+#DJANGO-SPAGHETTI-AND-MEATBALLS SETTINGS
+SPAGHETTI_SAUCE = {
+    'apps': ['auth', 'accounts', 'analytics', 'base', 'challenges', 'hosts', 'jobs', 'participants', 'web'],
+    'show_fields': True,
+}
```
Comment (deshraj) [inline settings/common.py]: Can you add this in dev.sample file?

File: evalai/urls.py [hunk 2]
```diff
@@ -56,3 +56,9 @@
                    include('participants.urls',
                            namespace='participants')),
                ]
+
+#DJANGO-SPAGHETTI-AND-MEATBALLS URLs
+urlpatterns += [
+  url(r'^dbschema/',
+   include('django_spaghetti.urls')),
+]
```
Comment (deshraj) [inline evalai/urls.py]: Make sure that url is only accessible in development environment.

File: settings/common.py (hunk 1 above)
Comment (RishabhJain2018) [inline settings/common.py]: okk sure..

File: evalai/urls.py (hunk 2 above)
Comment (RishabhJain2018) [inline evalai/urls.py]: yeah sure.

File: evalai/urls.py [hunk 3]
```diff
@@ -17,6 +17,7 @@
 from django.contrib import admin
 from accounts.views import ConfirmEmailView
 from django.views.generic.base import TemplateView
+from settings.dev import DEBUG
```
Comment (deshraj) [inline evalai/urls.py]: can you change this to `from django.conf import settings`?

File: evalai/urls.py [hunk 4]
```diff
@@ -56,3 +57,7 @@
                    include('participants.urls',
                            namespace='participants')),
                ]
+
+# DJANGO-SPAGHETTI-AND-MEATBALLS URLs available during development only.
+if DEBUG:
```
Comment (deshraj) [inline evalai/urls.py]: can you replace this with `if settings.DEBUG:`?

File: settings/common.py [hunk 5]
```diff
@@ -62,6 +62,7 @@
     'rest_framework.authtoken',
     'rest_framework',
     'rest_framework_docs',
+    'django_spaghetti',
```
Comment (deshraj) [inline settings/common.py]: Also, move it to `dev.sample` file

File: evalai/urls.py (hunk 3 above)
Comment (RishabhJain2018) [inline evalai/urls.py]: yeah sure.

File: evalai/urls.py (hunk 4 above)
Comment (RishabhJain2018) [inline evalai/urls.py]: okk..

File: settings/common.py (hunk 5 above)
Comment (RishabhJain2018) [inline settings/common.py]: yeah..

**Pull request merged**